staging/src/k8s.io/controller-manager/pkg/leadermigration/options/options_test.go

```diff
@@ -91,6 +91,24 @@ kind: LeaderMigrationConfiguration
 leaderName: test-leader-migration
 resourceLock: leases
 controllerLeaders: []
+`,
+			expectErr: false,
+			expectConfig: &config.LeaderMigrationConfiguration{
+				LeaderName:        "test-leader-migration",
+				ResourceLock:      "leases",
+				ControllerLeaders: []config.ControllerLeaderConfiguration{},
```

Comment:
Can we add a v1 test where we have a the default controllers + 1 custom (route, service, cloud-node-lifecycle and nodeipam)

Reply:
Sure! updated.